setup.py

```diff
@@ -19,18 +19,18 @@
     author="CFPB",
     author_email="tech@cfpb.gov",
     license="CCO",
-    version="2.2.1",
+    version="2.2.2",
```

Comment:
Since this is dropping support for Django 1.11, it should at least be 2.3.0, if not 3.0.0 (@chosak, thoughts?)